src/clojure/nrepl/server.clj

```diff
@@ -79,6 +79,12 @@
   (t/send transport (response-for msg :status #{:error :unknown-op :done} :op op)))
 
 (def default-middlewares
+  "middleware functions vars that are implicitly merged with any user-specified
```

Comment:
Please change this to:

`Middleware vars that are implicitly merged with any user-specified
   middlewares provided to `nrepl.server/default-handler`.`

Reply:
ok